docs/reference/query-dsl/combined-fields-query.asciidoc

```diff
@@ -5,14 +5,14 @@
 ++++
 
 The `combined_fields` query supports searching multiple text fields as if their
-contents had been indexed into one combined field. It takes a term-centric
+contents had been indexed into one combined field. {es} takes a term-centric
```

Comment:
It seems surprising to say "Elasticsearch takes a term-centric view of the query", because Elasticsearch as a whole does not usually do this. Maybe it'd be clearer to say "It takes a term-centric view of the input string", or "The query takes a term-centric view of the input string"?

Reply:
@jtibshirani, I'm +1 to that change.